boards/xtensa/esp32/esp32-devkitc/configs/psram_usrheap/defconfig

```diff
@@ -22,6 +22,7 @@ CONFIG_BUILTIN=y
 CONFIG_DEV_ZERO=y
 CONFIG_ESP32_SPIRAM=y
 CONFIG_ESP32_SPIRAM_USER_HEAP=y
+CONFIG_ESP32_SPI_FLASH_SUPPORT_PSRAM_STACK=y
```

Comment:
```suggestion
CONFIG_ESP32_SPIFLASH=y
CONFIG_ESP32_SPIRAM=y
CONFIG_ESP32_SPIRAM_USER_HEAP=y
CONFIG_ESP32_SPI_FLASH_SUPPORT_PSRAM_STACK=y
```
`ESP32_SPI_FLASH_SUPPORT_PSRAM_STACK` depends on `ESP32_SPIFLASH`, CI is complaining that the `defconfig` is invalid.